### ISSUE TITLE
Doesn't work in strict mode

### ISSUE THREAD

**Issue by Richie765** [loggr/loggr-nodejs#1]
Module can't be used in strict mode:

```
# node --use-strict my_script.js
/path/to/script/node_modules/loggr/lib/loggr.js:3
Loggr = {};
      ^

ReferenceError: Loggr is not defined
    at Object.<anonymous> (/path/to/script/node_modules/loggr/lib/loggr.js:3:7)
```

I don't think this global variable is really needed. Could it be made to work in strict mode?

**Comment by davideweaver** [comment]
The simple fix is to change the line to:

```
var Loggr = {};
```

Can you try that out in your application?

**Comment by Richie765** [comment]
That would indeed fix the problem but since that line is in the Loggr module, I think it should be done by you.

**Comment by davideweaver** [comment]
Right. Just wanted to make sure it would work for you. We'll update github and npm today. Thanks

**Comment by davideweaver** [comment]
The fix has been posted to npm (loggr@1.1.3)

**Comment by Richie765** [comment]
Tnx!
